app/models/service_provider_identity.rb

```diff
@@ -1,4 +1,35 @@
-# Joins Users to ServiceProviders
```

Comment:
this was a useful comment IMO, is there a way to preserve it?

Reply:
Looks like there is an option to put the annotation at the bottom of the file instead of the top. That would preserve existing top comments, and I doubt there are any existing comments after the class. Would also mean you wouldn't have to scroll past the comment to get to the code.